src/site/GlobalNavigation.css

```diff
@@ -0,0 +1,243 @@
+/* GLOBAL NAVIGATION */
+
+nav#dg_slide-menu a {
```

Comment:
this a better name for this is probably dg_site-menu

Reply:
Updated. Will push up with next commit.